# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/udev/hotfix-clock-group.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- systemd-258/rules.d/50-udev-default.rules.in.vanilla	2025-09-29 11:58:36.662857952 +0200
+++ systemd-258/rules.d/50-udev-default.rules.in	2025-09-29 11:58:49.345887663 +0200
@@ -10,7 +10,7 @@
 
 SUBSYSTEM=="virtio-ports", KERNEL=="vport*", ATTR{name}=="?*", SYMLINK+="virtio-ports/$attr{name}"
 
-SUBSYSTEM=="rtc", GROUP="clock", MODE="0660"
+SUBSYSTEM=="rtc", GROUP="root", MODE="0660"
 # select "system RTC" or just use the first one
 SUBSYSTEM=="rtc", ATTR{hctosys}=="1", SYMLINK+="rtc"
 SUBSYSTEM=="rtc", KERNEL=="rtc0", SYMLINK+="rtc", OPTIONS+="link_priority=-100"
@@ -28,7 +28,7 @@
 
 SUBSYSTEM=="net", IMPORT{builtin}="net_driver"
 
-SUBSYSTEM=="ptp", GROUP="clock", MODE="0660"
+SUBSYSTEM=="ptp", GROUP="root", MODE="0660"
 SUBSYSTEM=="ptp", ATTR{clock_name}=="KVM virtual PTP", SYMLINK+="ptp_kvm"
 SUBSYSTEM=="ptp", ATTR{clock_name}=="hyperv", SYMLINK+="ptp_hyperv"
 
